{
    "name": "libcusolver",
    "ecosystem": "pypi",
    "campaign": "2025-10-bugdotexe",
    "affected_versions": [
        "2.3.2"
    ],
    "campaign_assigned_at": "2025-10-29T22:06:04.026253+00:00",
    "campaign_data_modified_at": "2025-10-29T22:06:04.026268+00:00"
}